version/version.go

```diff
@@ -1,4 +1,42 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2016 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
 package version
 
-// Version string, k6 uses semantic versioning (http://semver.org/)
-const Version = "0.17.1"
+import "fmt"
+
+// Version represents a sember - k6 uses semantic versioning (http://semver.org/)
+type Version struct {
+	Major uint
+	Minor uint
+	Patch uint
+}
+
+// Current represents the current version and can be shared across packages
+var Current Version
+
+func init() {
+	Current = Version{Major: 0, Minor: 18, Patch: 2}
```

Comment:
No need to do this in an init, this is valid:

```go
var Current = Version{Major: 0, Minor: 18, Patch: 2}
```

Or even this, if you want:

```go
var Current = Version{0, 18, 2}
```

Reply:
yes, good point